src/qtoolkit/manager.py

```diff
@@ -69,12 +77,12 @@ def get_environment_setup(self, env_config) -> str:
         if env_config:
             env_setup = []
             if "modules" in env_config:
-                env_setup.append("module purge")
-                for mod in env_config["modules"]:
-                    env_setup.append(f"module load {mod}")
+                env_setup += [f"module load {mod}" for mod in env_config["modules"]]
```

Comment:
it seems that in this change the `env_setup.append("module purge")` was lost. Am I wrong?